### DISCUSSION TITLE
Is it possible to develop a vuesx dedicated to vue and support vue's template syntax in jsx?

### DISCUSSION THREAD

**Discussion by alamhubb** [vuejs/rfcs#655] [General]
I really like the syntax of vue's template, but jsx does not support it. I want to use template in the jsx way, but I haven't found any good way. Can the official introduce a vuesx syntax?

**Comment by zhiyuanzmj** [comment]
https://vue-macros.dev/features/jsx-directive.html
<img width="733" alt="image" src="https://github.com/vuejs/rfcs/assets/32807958/e3a202d1-5b29-4f16-8d38-92662ce82ca7">

**Reply by jacekkarczmarczyk** [reply]
It's still different that Vue template syntax

**Reply by zhiyuanzmj** [reply]
Thank you for your comments. I'm just curious. Can you give provide some examples ?
